run_registration.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash -e
-BROWSER=chrome
-BROWSER_TYPE=$1
-ENV=$2
+
+ENV=${ENV:-$1}
+BROWSER=${BROWSER:-$2}
 
 DRIVER=
 
-if [ "$BROWSER" = "headless-chrome" ] || [ "$BROWSER" = "chrome" ] ; then
+if  [ "$BROWSER" = "chrome" ] ; then
     DRIVER="-Dwebdriver.chrome.driver=/usr/local/bin/chromedriver"
 elif [ "$BROWSER" = "firefox" ]; then
     DRIVER="-Dwebdriver.gecko.driver=/usr/local/bin/geckodriver"
 fi
 
 
 # Scalafmt checks have been separated from the test command to avoid OutOfMemoryError in Jenkins
-sbt scalafmtCheckAll scalafmtSbtCheck
+# sbt scalafmtCheckAll scalafmtSbtCheck
 
-sbt -Dbrowser="${BROWSER_TYPE:=$BROWSER}" -Denvironment="${ENV:=local}" "testOnly uk.gov.hmrc.test.ui.cucumber.runner.RunRegistration"
+sbt -Dbrowser=$BROWSER -Denvironment=$ENV $DRIVER "testOnly uk.gov.hmrc.test.ui.cucumber.runner.RunRegistration"
```

Comment:
Can we try and keep the defaults of `chrome` and `local` please? This is handy when wanting to run the tests quickly, can't see a reason why it shouldn't work?

Reply:
changed.